{{ cookiecutter.app_name }}/src/webapp/webapp/settings.py

```diff
@@ -0,0 +1,123 @@
+"""
+Django settings for webapp project.
+
+Generated by 'django-admin startproject' using Django 4.2.1.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.2/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.2/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.2/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'django-insecure-86kqbg4s&53!bmq5)pd0hg@a%7b1pn_7&yw(tljy=xh12*!_u$'  # noqa: E501
```

Comment:
This should probably be generated by the template, rather than hard-coded with a warning.

Reply:
we can also strip back the code to not include those generate by Django, anyway, will close this PR